applications/scripts/deep_consensus/deep_consensus.py

```diff
@@ -0,0 +1,313 @@
+                      outParticlesPath, gpuToUse, numberOfThreads):
+        '''
+            outParticlesPath= self._getPath("particles.xmd")
+            posTestDict, negTestDict, predictDict: { fnameToMetadata:  weight:int }
+        ''' 
+        if gpuToUse >= 0:
+            numberOfThreads = None
+        else:
+            gpuToUse = None
+            if numberOfThreads < 0:
+                import multiprocessing          
+                numberOfThreads = multiprocessing.cpu_count()
+
+        updateEnviron(gpuToUse)
+
+        predictDataManager = DataManager(posSetDict=predictDict, negSetDict=None)
+        dataShape_nTrue_numModels = loadNetShape(netDataPath)
+        if dataShape_nTrue_numModels:
+          numModels= dataShape_nTrue_numModels[-1]
+        else:
+          numModels=1
+        try:
+            nnet = DeepTFSupervised(numberOfThreads=numberOfThreads,
+                                    rootPath=netDataPath,
+                                    numberOfModels=numModels)
+        except tf_intarnalError as e:
+            if e._error_code == 13:
+                raise Exception("Out of GPU Memory. GPU # %d" % (gpuToUse))
+            else:
+                raise e
+
+        y_pred, label_Id_dataSetNumIterator = nnet.predictNet(predictDataManager)
+
+        metadataPosList, metadataNegList = predictDataManager.getMetadata(None)
+        for score, (isPositive, mdId, dataSetNumber) in zip(y_pred, label_Id_dataSetNumIterator):
+            if isPositive==True:
+                metadataPosList[dataSetNumber].setValue( xmippLib.MDL_ZSCORE_DEEPLEARNING1, score, mdId)
+            else:
+                metadataNegList[dataSetNumber].setValue( xmippLib.MDL_ZSCORE_DEEPLEARNING1, score, mdId)
+
+        assert len(metadataPosList) == 1, \
+                "Error, predict setOfParticles must contain one single object"
+
+        metadataPosList[0].write(outParticlesPath)
+        if posTestDict and negTestDict:
+            testDataManager = DataManager(posSetDict=posTestDict,
+                                          negSetDict=negTestDict,
+                                          validationFraction=0)
+            print("Evaluating test set")
+            global_auc, global_acc, y_labels, y_pred_all = nnet.evaluateNet(testDataManager)
+            if WRITE_TEST_SCORES:
+              with open(os.path.join(netDataPath, "testPredictions.txt"), "w") as f:
+                  f.write("label score\n")
+                  for l, s in zip(y_labels, y_pred_all):
+                      f.write("%d %f\n" % (l, s))
+
+def updateEnviron(gpuNum=None):
+  """ Create the needed environment for TensorFlow programs. """
+  print("updating environ to select gpu %s"%(gpuNum) )
+  if not gpuNum is None:
+    os.environ['LD_LIBRARY_PATH']= os.environ['CUDA_LIB']+":"+os.environ['CUDA_HOME']+"/extras/CUPTI/lib64"
+    os.environ['LD_LIBRARY_PATH']= os.environ['CUDA_LIB']
```

Comment:
`CUDA_HOME` doesn't exist in the default Scipion/Xmipp enviroment configuration. In fact, `CUDA_LIB` is already added to the `LD_LIBRARY_PATH`. On the other hand, I run the protocol without `os.environ['CUDA_HOME']+"/extras/CUPTI/lib64"` in the `LD_LIBRARY_PATH` and it run properly. If it is needed in somehow, we should putted in the `xmipp-bundle/src/scipion-em-xmipp/xmipp3/__init__.py:L53 [Plugin.getEnviron()]`

Reply:
I  defined CUDA_HOME in scipion/config/scipion.conf

CUDA_HOME = /home/rsanchez/app/cuda-8.0
CUDA_LIB = %(CUDA_HOME)s/lib64
CUDA_BIN = %(CUDA_HOME)s/bin

I do not know if it should be removed.
Btw, the os.environ LD_LIBRARY_PATH contains both lib64 and and extras/CUPTI/lib64 so this update can be removed